src/TuxArchitectStores.js

```diff
@@ -65,31 +65,53 @@ ArchitectChain.prototype.itOutputs = function (output) {
 //_getRegisterId FUNCTION: internal function,for getting the store __registerId__ from either the passed in store or the store mapped to the passed in input
```

Comment:
Change comment to ...the store \_\_registerId__ or set of \_\_registerId__'s...  also change comment to ...or the store or set of stores...